notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    if os.getenv("IS_TESTING"):
</pre><p>Why does this differ for testing?</p><p> </p><p>Also, please include the explanation to the user reading this in the notebook itself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
In short, we can delete this line and only use the first one, located in `https://github.com...`.

The testing environment has an issue to read the file that start with "gs://" since Hadoop is not configured in the testing env. It requires additional configs but since it is a testing environment, but I didn't think that's necessary.

Reply:
<p>Thanks Hyunuk for clarifying. I also spent some time trying to get the GCS connector to work in the test environment but ran into some issues given how Spark is set up via Pip. This may be relevant in future notebook samples but for now this workaround should suffice.</p>